spec/factories.rb

```diff
@@ -536,6 +536,14 @@
       reject_by_default_days { 40 }
     end
 
+    trait :with_scheduled_interview do
```

Comment:
We should make sure this looks like it's been through the awaiting_provider decision state first: Can we just add
```
awaiting_provider_decision
```
at the top to handle that for us?